tests/0128/parser-error.txt

```diff
@@ -0,0 +1 @@
+ExpectedToken(["an identifier"], Some("s"), (5, 25)) -> Parse error: unexpected token `s`, expecting an identifier on line 5 column 25
```

Comment:
Is there some way we could improve this error? The _real_ error is that there's no type defined for the prop/param.

Reply:
Yes, next PR will focus more on properties and promoted properties 🤞